components/kyma-environment-broker/internal/storage/driver/memory/instance.go

```diff
@@ -153,15 +154,24 @@ func (s *instances) GetByID(instanceID string) (*internal.Instance, error) {
 		return nil, dberr.NotFound("instance with id %s not exist", instanceID)
 	}
 
+	marshaled, err := json.Marshal(inst)
+	unmarshaledInstance := internal.Instance{}
+	err = json.Unmarshal(marshaled, &unmarshaledInstance)
+
```

Comment:
What is the point?

Reply:
Added a comment explaining the reasoning behind the code.